uaclient/security_status.py

```diff
@@ -120,27 +125,37 @@ def security_status(cfg: UAConfig) -> Dict[str, Any]:
     installed_packages = [package for package in cache if package.is_installed]
     summary["num_installed_packages"] = len(installed_packages)
 
-    security_upgradable_versions = filter_security_updates(installed_packages)
+    package_count = {"esm-infra": 0, "esm-apps": 0}
+    update_count = {"esm-infra": 0, "esm-apps": 0, "standard-security": 0}
 
-    package_count = {"esm-infra": 0, "esm-apps": 0, "standard-security": 0}
+    for package in installed_packages:
+        if is_esm_package(package, "esm-apps"):
+            package_count["esm-apps"] += 1
+        if is_esm_package(package, "esm-infra"):
+            package_count["esm-infra"] += 1
```

Comment:
If the package appear on both pockets for some reason, we would be counting it twice, right ?

Reply:
Yes, but it is ok. The information we are providing is: you have |X| packages which are available in ESM-Infra, and |Y| packages which are available in ESM-Apps, but X and Y are not mutually exclusive - the same version can be in both